extensions/ql-vscode/src/databases/config/db-config-store.ts

```diff
@@ -134,15 +134,13 @@ export class DbConfigStore extends DisposableObject {
       throw Error("Cannot add local list if config is not loaded");
     }
 
-    if (listName === "") {
-      throw Error("List name cannot be empty");
-    }
+    this.validateLocalListName(listName);
 
     if (this.doesLocalListExist(listName)) {
       throw Error(`A local list with the name '${listName}' already exists`);
     }
```

Comment:
This is now included in the validation function, so can we delete it here?

Reply:
Oops I think that was a git fail on my behalf, thanks for catching!